docs/CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@ If you find an issue with this package, please create an issue in GitHub. If you
 
 ## Adding new functionality or fixing relevant bugs
 
-If you are adding a new endpoint, make sure to update COVERAGE.md where we keep a list of the TFC APIs that this SDK supports.
+If you are adding a new endpoint, make sure to update the coverage list in README.md where we keep a list of the TFC APIs that this SDK supports.
```

Comment:
Could we add a link to the coverage section in the readme (and perhaps have an anchor in the readme)?